lms/static/sass/multicourse/_dashboard.scss

```diff
@@ -350,357 +285,401 @@
         }
       }
     }
+  }
+}
 
-    // ====================
+// +Dashboard - Course
+// ==================== 
+.dashboard .my-courses {
 
-    // UI: individual course item
-    .course {
-      @include box-sizing(box);
-      @include transition(all 0.15s linear 0s);
+  // UI: individual course item
+  .course {
+    @include box-sizing(box);
+    @include transition(all 0.15s linear 0s);
+    @include clearfix();
+    @extend %ui-depth2;
+    position: relative;
+    margin: ($baseline/2);
+
+    .details {
       @include clearfix();
-      @extend %ui-depth2;
-      position: relative;
 
-      .cover {
-        @include box-sizing(border-box);
-        @include transition(all 0.15s linear 0s);
-        overflow: hidden;
-        position: relative;
+      .wrapper-course-image {
         @include float(left);
-        height: 100%;
-        max-height: 100%;
-        width: 200px;
-        height: 120px;
-        margin: 0;
-        border-radius: ($baseline/10);
-        border: 1px solid $dashboard-course-cover-border;
-        border-bottom: 4px solid $dashboard-course-cover-border;
-        padding: ($baseline/10);
+        @include margin-right(flex-gutter());
+        width: flex-grid(3);
 
-        img {
-          width: 100%;
-          min-height: 100%;
-        }
-      }
+        .cover {
+          @include box-sizing(border-box);
+          @include transition(all 0.15s linear 0s);
+          @include float(left);
+          overflow: hidden;
+          position: relative;
+          max-height: 120px;
+          border-radius: ($baseline/5);
+          border: 1px solid $dashboard-course-cover-border;
+          border-bottom: 4px solid $dashboard-course-cover-border;
 
-      .info {
-        @include clearfix();
-        @include padding(0, 10px, 0, 230px);
+          .course-image {
+            max-width: 100%;
```

Comment:
after testing this on firefox, this needs to be changed back to width instead of max-width. I'm not sure why yet but Firefox renders the full width of the image if you use max-width instead of scaling the image.